backend/.gitignore

```diff
@@ -1,2 +1 @@
 node_modules
-.env
```

Comment:
.env files should not be added to github : they may contain passwords. you should keep them ignored